### PR TITLE
Add tRNS PNG test

### PR DESCRIPTION
**Pull request by ProgramMax**
The PNG spec Third Edition clarified that high bits of a tRNS chunk should be ignored on a lower bit depth image. Previous editions were unclear, which caused divergent behavior.

This commits adds a PNG image which contains a tRNS chunk with non-zero values in its high bits. This commit also adds a test to verify these high bits are ignored.

### CODE REVIEW

**Comment by ProgramMax** [comment]
I would add @svgeesus as a reviewer but I can't seem to.

This should land after https://github.com/web-platform-tests/wpt/pull/37187 because that other pull request adds the META.yml file.

This test is the result of https://github.com/w3c/PNG-spec/issues/29 . The issue at hand is PNG stores tRNS colors as 16-bit-per-channel, even when the image is 8-bit. In that case, the high bits of each color channel should be ignored. Firefox will treat the image as if it contains an error if those high bits aren't zero. This likely came from poor wording in older versions of the PNG spec. But since we're improving that wording, we should also add a test to enforce it.

File: png/trns-chunk.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<title>PNG test: cICP chunk</title>
```
Comment (svgeesus) [inline png/trns-chunk.html:2]: Please update the title :)

Comment (svgeesus) [reply png/trns-chunk.html:2]: Add 
```
<link rel="help" href="https://w3c.github.io/PNG-spec/#11tRNS">
```

File: png/trns-chunk.html [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    _assertSame(pixel.length, pixel_expected.length, "pixel.length", "pixel_expected.length");
+    // Ignore the color channel comparisons.
+    // The image itself stores 255 red, 0 green, 0 blue.
+    // But there may not be a requirement that this be preserved
+    // while decoding or through canvas processing.
+    assert_approx_equals(pixel[3], pixel_expected[3], epsilon);
+  });
+};
+img.src = "/images/trns.png";
```
Comment (svgeesus) [inline png/trns-chunk.html:35]: Should that be a relative link?

**Review by svgeesus — CHANGES_REQUESTED** [review]

File: png/trns-chunk.html (hunk 1 above)
Comment (ProgramMax) [reply png/trns-chunk.html]: Done.

File: png/trns-chunk.html (hunk 2 above)
Comment (ProgramMax) [reply png/trns-chunk.html]: Done.

**Review by svgeesus — APPROVED** [review]

**Pull request merged**